team_delta.js

```diff
@@ -1 +1,26 @@
 // Fetch all routes for the airline and insert into #delta
+const getFlight = async (from, to, resourse) => {
+  const response = await fetch(resourse);
+  const dataJson = await response.json();
+  const searchEl = document.createElement("div");
```

Comment:
Try to separate the DOM creation from the fetch logic (make a new function to create the dom elements).